README.md

```diff
@@ -50,7 +50,10 @@
 - **Follow HTML and CSS best practice**
 - **Display Text**
 - **Follow GitHub flow**
-
+- **Display Cards**
+- **Hoover Buttons**
+- **Mobile Version**
+- **Desktop Version**
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
```

Comment:
Please remove Live Demo link and Live Demo heading  if you didn't deploy this project. Currently, the link goes to another project. It is important for having a professional README file.

Reply:
I Deplyed my page and put the right URL.
thanks